cabal-install/src/Distribution/Client/ProjectConfig/Legacy.hs

```diff
@@ -203,9 +204,12 @@ import System.FilePath (isAbsolute, isPathSeparator, makeValid, splitFileName, (
 import Text.PrettyPrint
   ( Doc
   , render
+  , semi
+  , text
+  , vcat
   , ($+$)
   )
-import qualified Text.PrettyPrint as Disp
+import qualified Text.PrettyPrint as Disp (empty, int, render, text)
```

Comment:
`text` again? (see above)

Reply:
Same goes for `render`. Actually only `empty` needs to be used qualified.